app/models/legacy_appeal.rb

```diff
@@ -391,41 +401,68 @@ def scheduled_hearings
-  #   In response to COVID, the appellant has the option of changing their hearing
-  #   preference if they were scheduled for a travel board hearing. This method captures
-  #   if a travel board hearing request type was overridden in Caseflow.
-  def sanitized_hearing_request_type
-    if changed_request_type.present?
-      case changed_request_type
-      when HearingDay::REQUEST_TYPES[:video]
-        return :video
-      when HearingDay::REQUEST_TYPES[:virtual]
-        return :virtual
-      else
-        fail InvalidChangedRequestType, "\"#{changed_request_type}\" is not a valid request type."
-      end
-    end
-
-    original_hearing_request_type
+  # from a travel board hearing
+  # This method cleans all of these issues up to return a sanitized version of the original type requested by Appellant.
+  # Flag readable is mostly used by serializers
+  def original_hearing_request_type(readable: false)
+    original_hearing_request_type = case hearing_request_type
+                                    when :central_office
+                                      :central_office
+                                    when :travel_board
+                                      video_hearing_requested ? :video : :travel_board
+                                    end
+    readable ? READABLE_HEARING_REQUEST_TYPES[original_hearing_request_type] : original_hearing_request_type
   end
 
-  def readable_hearing_request_type
-    READABLE_HEARING_REQUEST_TYPES[sanitized_hearing_request_type]
-  end
+  # [Sept. 2020]:
+  #   In response to COVID, the appellant has the option of changing their hearing
+  #   preference if they were scheduled for a travel board hearing.
+  #   This method captures if a travel board hearing request type was overridden in Caseflow.
+  # In general, this method returns the current hearing request type which could be dervied
+  # from `change_request_type` or VACOLS `hearing_request_type`
+  # Flag `readable` is mostly used by serializers
+  def current_hearing_request_type(readable: false)
+    current_hearing_request_type = if changed_request_type.present?
+                                     sanitized_changed_request_type(changed_request_type)
+                                   else
+                                     original_hearing_request_type
+                                   end
+    readable ? READABLE_HEARING_REQUEST_TYPES[current_hearing_request_type] : current_hearing_request_type
+  end
+
+  # if `change_hearing_request` is populated meaning the hearing request type was changed, then return what the
+  # previous hearing request type was. Use paper trail event to derive previous type in the case the type was changed
+  # multple times.
+  # Flag `readable` is mostly used `ChangeHearingRequestTypeTask`
+  def previous_hearing_request_type(readable: false)
+    # Example of diff: {"changed_request_type"=>[nil, "R"]}
+    previous_unsanitized_type = latest_appeal_event.diff["changed_request_type"].first
```

Comment:
```suggestion
    previous_unsanitized_type = latest_appeal_event&.diff["changed_request_type"]&.first
```
So we can call `previous_hearing_request_type` on appeals without a papertrail without getting a `NoMethodError`.